Src/GoogleApis.Auth/OAuth2/UserCredential.cs

```diff
@@ -133,6 +124,23 @@ public void Initialize(ConfigurableHttpClient httpClient)
 
         #endregion
 
+        #region ITokenAccess implementation
+
+        public virtual async Task<string> GetAccessTokenForRequestAsync(string authUri = null, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            if (Token.IsExpired(flow.Clock))

```

Comment:
I think for user credential (unlike service credential) the Token is never null. It gets assigned in the constructor. 
Again, this is preexisting code.